### PR TITLE
vendor: update opencontainers/selinux v1.3.3

### PR DESCRIPTION
**Pull request by thaJeztah**
full diff: https://github.com/opencontainers/selinux/compare/v1.3.1...v1.3.3

- go-selinux/SetKeyLabel: fix for RHEL7 kernels
- getSELinuxfs: optimize/simplify using sync.Once
- xattr: use x/sys/unix, simplify
- Use /proc/thread-self if available
- Fix EnforceMode, SetEnforceMode, and SecurityCheckContext

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
opened as draft, because I just saw a comment that this broke 32-bit platforms (arm); https://github.com/opencontainers/selinux/pull/64#issuecomment-592047943

**Comment by thaJeztah** [comment]
^ the above is fixed on master through https://github.com/opencontainers/selinux/pull/71, but not yet in a release

**Comment by thaJeztah** [comment]
v1.3.3 was tagged with a fix for the regression; updated this PR, this should now be ready for review @kolyshkin

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go [hunk 1]
```diff
@@ -254,10 +247,17 @@ func getSELinuxPolicyRoot() string {
...
 
-func isProcHandle(fh *os.File) (bool, error) {
+func isProcHandle(fh *os.File) error {
 	var buf unix.Statfs_t
 	err := unix.Fstatfs(int(fh.Fd()), &buf)
-	return buf.Type == unix.PROC_SUPER_MAGIC, err
+	if err != nil {
+		return fmt.Errorf("statfs(%q) failed: %v", fh.Name(), err)
+	}
+	if buf.Type != unix.PROC_SUPER_MAGIC {
```
Comment (thaJeztah) [inline vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: @kolyshkin I think this should also be treated the same?

```suggestion
	if uint32(buf.Type) != uint32(unix.PROC_SUPER_MAGIC) {
```

Comment (thaJeztah) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: oh, nevermind; looks like this `const` is not problematic

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Failure on powerpc; https://ci-next.docker.com/public/job/moby/job/PR-40597/3/

```
[2020-02-28T18:53:39.836Z] === FAIL: ppc64le.integration.build TestBuildWithHugeFile (81.84s)
[2020-02-28T18:53:39.836Z]     build_test.go:519: assertion failed: string "{\"stream\":\"Step 1/2 : FROM busybox\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e f4279da41337\\n\"}\r\n{\"stream\":\"Step 2/2 : RUN for g in $(seq 0 8); do dd if=/dev/urandom of=rnd bs=1K count=1 seek=$((1024*1024*g)) status=none; done \\u0026\\u0026     ls -la rnd \\u0026\\u0026 du -sk rnd\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in 9fe3bd4446da\\n\"}\r\n{\"stream\":\"-rw-r--r--    1 root     root     8589935616 Feb 28 18:52 rnd\\n\"}\r\n{\"stream\":\"36\\trnd\\n\"}\r\n{\"stream\":\"Removing intermediate container 9fe3bd4446da\\n\"}\r\n{\"errorDetail\":{\"message\":\"Error processing tar file(exit status 1): write /rnd: no space left on device\"},\"error\":\"Error processing tar file(exit status 1): write /rnd: no space left on device\"}\r\n" does not contain "Successfully built"
```

Should not be related, but kicking CI again

**Comment by thaJeztah** [comment]
merging; this is green now

**Pull request merged**